build.gradle

```diff
@@ -25,7 +25,7 @@ ext {
 
   androidXVersions = [
     annotation            : '1.0.1',
-    appCompat             : '1.1.0',
+    appCompat             : '1.4.0-alpha02', // Need AppCompatDialogFragment constructor that takes a layout ID
```

Comment:
We should not use the alpha version of a library in this project.

Reply:
The functionality needed for this PR will only available in AppCompat 1.4.0, so we will need to way for the stable version of it.